src/index.mjs

```diff
@@ -51,15 +52,22 @@ const NAME = 'SizePlugin';
  * @param {Object} options
  * @param {string} [options.pattern] minimatch pattern of files to track
  * @param {string} [options.exclude] minimatch pattern of files NOT to track
+ * @param {string} [options.filename] file name to save filesizes to disk
+ * @param {string} [options.publish] option to publish filesizes to size-plugin-store
+ * @param {string} [options.writeFile] option to save filesizes to disk
  * @param {function} [options.stripHash] custom function to remove/normalize hashed filenames for comparison
  * @param {(item:Item)=>string?} [options.decorateItem] custom function to decorate items
  * @param {(data:Data)=>string?} [options.decorateAfter] custom function to decorate all output
  */
 export default class SizePlugin {
-	constructor (options) {
+	constructor(options) {
 		this.options = options || {};
 		this.pattern = this.options.pattern || '**/*.{mjs,js,css,html}';
 		this.exclude = this.options.exclude;
+		this.options.filename = this.options.filename || 'size-plugin.json';
+		this.options.publish = this.options.publish !== false;
```

Comment:
Best to avoid surprising folks with a remote connection they didn't opt into.

```suggestion
		this.options.publish = this.options.publish === true;
```

Reply:
yup, that makes sense. I was biased for the bot.